### ISSUE TITLE
Website does not show any information

### ISSUE THREAD

**Issue by zakrok** [JoanMVPopov/perfumedia#12]
Is this a bug? Website does not show any information about the fragrances present!

![image](https://github.com/user-attachments/assets/22dae09c-ccc4-463b-93c2-67ddd3a1089c)

Should I be using a different browser perhaps? I did this on 
Name 	Firefox
Version 	128.0.3-2
Build ID 	20240803114312
User Agent 	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
OS 	Windows_NT 10.0 19045

**Comment by denislav-d** [comment]
I have the same issue. @Zakrok09 @JoanMVPopov
Using Arc 1.73.0 (56815)
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
OS: MacOS Sonoma 14.7 (23H124)

**Comment by zakrok** [comment]
Issue doesn't seem to be fixed? Am I missing something? Maybe it works for @denislav-d ?

**Comment by denislav-d** [comment]
Nope, still doesn’t work for me. Looking forward to  getting a fix soon! 🙏

**Comment by zakrok** [comment]
@denislav-d the website finally has information! time to use my knowledge in Calculus III to choose a perfume 💯 🚀

**Comment by denislav-d** [comment]
@Zakrok09 it is amazing!!! I got hungry after looking at all of these pie charts 🥧

**Comment by JoanMVPopov** [comment]
I want to extend a big thank you to Kiril Panayotov and Denislav Dimitrov for inspiring and motivating me to bring Perfumedia to life. 🙌

Your energy, feedback, and belief in creative ideas were the push I needed to get this project off the ground. It's exciting to work on something that blends technology, storytelling, and passion — and it wouldn’t have started without that initial spark from our conversations.

Looking forward to what comes next!

#gratitude #perfumedia #startupjourney #creativity #inspiration #thankyou

**Comment by zakrok** [comment]
AI cannot recreate such a beautifully written mesage
